mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -262,6 +262,10 @@ region are guaranteed to support the same version.
     to say "these are the _only_ node pools associated with this cluster", use the
     [google_container_node_pool](container_node_pool.html) resource instead of this property.
 
+* `node_pool_auto_config` - (Optional) Node pool configs that apply to auto-provisioned node pools in
```

Comment:
Here & below need to be marked as beta-only

```suggestion
* `node_pool_auto_config` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Node pool configs that apply to auto-provisioned node pools in
```

Reply:
Ah right, of course. Got excited and missed that bit. Fixed now